frontend/src/__tests__/cypress/cypress/e2e/pipelines/PipelinesTopology.cy.ts

```diff
@@ -153,4 +210,71 @@ describe('Pipeline topology', () => {
         .contains('conditional-execution-pipeline');
     });
   });
+  describe('Pipelines logs', () => {
+    it('Test Logs tab', () => {
+      initIntercepts();
+      pipelineRunDetails.visit('test-project', 'test-pipeline-run-id');
+      // test whether you can access the pipeline run right drawer
+      pipelineRunDetails.findTaskNode('flip-coin').click();
```

Comment:
Curious why this was one test block -- and not individual tests with specific goals and using something like `beforeEach` to manage interceptors, visiting, and getting into the sidebar... you strung your entire test suite together and had to come up with comments instead of just test names stating the goal of the test

Reply:
added, thanks